qemu/tests/balloon_stress.py

```diff
@@ -16,14 +16,53 @@ def run(test, params, env):
     2) enable driver verifier in guest
     3) reboot guest (optional)
     4) check device using right driver in guest.
-    5) play online video in guest
+    5) play video in background repeatly
     6) balloon memory in monitor in loop
-    7) check vm alive
+    7) clean the video
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+    def check_video_status(vm):
```

Comment:
test won't raise when error happen in thread, so I check process in memory balloon loop